### PR TITLE
.cci.jenkinsfile: change path for download-overrides.py

### PR DESCRIPTION
**Pull request by c4rt0**
Once below PR's, requesting the move of download-overrides.py
has been merged, the file path will change to modified. 

See: https://github.com/coreos/coreos-assembler/pull/3510
See: https://github.com/coreos/fedora-coreos-releng-automation/pull/176

Issue:

See: https://github.com/coreos/fedora-coreos-tracker/issues/1185

### CODE REVIEW

File: .cci.jenkinsfile [hunk 1]
```diff
@@ -12,7 +12,7 @@ cosaPod(cpus: 4, memory: "9Gi") {
     shwrap("""
         mkdir -p /srv/coreos && cd /srv/coreos
         cosa init ${env.WORKSPACE}/config
-        curl -LO https://raw.githubusercontent.com/coreos/fedora-coreos-releng-automation/main/scripts/download-overrides.py
+        curl -LO https://raw.githubusercontent.com/coreos/coreos-assembler/main/src/download-overrides.py
```
Comment (dustymabe) [inline .cci.jenkinsfile]: once this is in COSA we won't need to curl since we are already inside a cosa container.

Comment (c4rt0) [reply .cci.jenkinsfile]: > once this is in COSA we won't need to curl since we are already inside a cosa container.

Oh, true... thanks.

Comment (dustymabe) [reply .cci.jenkinsfile]: but.. we still need to remove the curl and possibly change the path in the `python3` call below.

Comment (c4rt0) [reply .cci.jenkinsfile]: > but.. we still need to remove the curl and possibly change the path in the `python3` call below.

The GitHub emoji selection should seriously be more abundant.

:adamwithclosedeyesholdingforehead

Understood.

**Comment by c4rt0** [comment]
Honestly... I'm still trying to figure out where the `download-overrides.py` will be placed.

File: .cci.jenkinsfile [hunk 2]
```diff
@@ -12,8 +12,9 @@ cosaPod(cpus: 4, memory: "9Gi") {
     shwrap("""
         mkdir -p /srv/coreos && cd /srv/coreos
         cosa init ${env.WORKSPACE}/config
-        curl -LO https://raw.githubusercontent.com/coreos/fedora-coreos-releng-automation/main/scripts/download-overrides.py
-        python3 download-overrides.py
+        pwd
+        ls -al ${env.WORKSPACE}
+        python3 ${env.WORKSPACE}/usr/lib/coreos-assembler/download-overrides.py
```
Comment (dustymabe) [inline .cci.jenkinsfile]:
```suggestion
        python3 /usr/lib/coreos-assembler/download-overrides.py
```

**Comment by dustymabe** [comment]
CI is going to fail until https://github.com/coreos/coreos-assembler/pull/3510 merges

**Review by dustymabe — APPROVED** [review]

**Pull request merged**